pkg/limits/frontend/service.go

```diff
@@ -49,15 +52,41 @@ type RingIngestLimitsService struct {
 	pool *ring_client.Pool
 
 	limits Limits
+
+	// metrics
+	tenantExceedsLimits         *prometheus.CounterVec
```

Comment:
We tend to use `Metrics` structs to hold our metrics. I would suggest we do it here too. For example, https://github.com/search?q=repo%3Agrafana%2Floki+type+metrics&type=code.